CSS/style.css

```diff
@@ -1,3 +1,47 @@
 @import url("./basic-styling/color.css");
 @import url("./basic-styling/reset.css");
 @import url("./basic-styling/typography.css");
+
+@import url("./Components/index.css");
+
+/* Components Css */
+.container {
+  min-height: 100vh;
+  width: 80%;
+  margin: auto;
```

Comment:
margin: 0 auto; will center it.

auto at top and bottom does not work as expected always